src/utils/formatters.ts

```diff
@@ -190,6 +190,31 @@ module Plottable {
     }
 
     /**
+     * Transforms the Plottable TimeInterval into a d3 time interval equivalent
+     */
+    export function timeIntervalToD3Time(timeInterval: TimeInterval) {
+      switch (timeInterval) {
+      case TimeInterval.second:
+        return d3.time.second;
+      case TimeInterval.minute:
+        return d3.time.minute;
+      case TimeInterval.hour:
+        return d3.time.hour;
+      case TimeInterval.day:
+        return d3.time.day;
+      case TimeInterval.week:
+        return d3.time.week;
+      case TimeInterval.month:
+        return d3.time.month;
+      case TimeInterval.year:
+        return d3.time.year;
+      default:
+        Utils.Methods.warn("TimeInterval specified does not exist");
+        return d3.time.year;
```

Comment:
The reason this default is here is the rendering logic. If this function fails and throws an exception, the exception is going to be caught by https://github.com/palantir/plottable/blob/timeD3/src/core/renderController.ts#L95 and the renderer is going to loop infinitely. We need to either keep this default or fix the renderer. 

In order to test the renderer fix, leave out lines 209 and 210 (the year case).